test/integration/examples/apiserver_test.go

```diff
@@ -154,9 +155,13 @@ func TestAggregatedAPIServer(t *testing.T) {
 			t.Log(err)
 			return false, nil
 		}
-		if _, err := kubeClient.Discovery().ServerVersion(); err != nil {
+
+		healthStatus := 0
+		kubeClient.Discovery().RESTClient().Get().AbsPath("/healthz").Do().StatusCode(&healthStatus)
```

Comment:
why this change?

Reply:
Just because it is more strict? (as the listener will be active here already)

Reply:
Yes

Reply:
ok